lib/components/util/formatted-mode.js

```diff
@@ -0,0 +1,63 @@
+import { FormattedMessage } from 'react-intl'
```

Comment:
Is this new in this PR? otherwise we should probably merge dev into this first to remove all changes already merged.

Reply:
Yep this is new in this PR